opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -221,6 +233,10 @@ class DefaultSpan(Span):
     """
 
     def __init__(self, context: "SpanContext") -> None:
+        # The only thing the ``__init__()`` method on the base class does is
+        # initialize self.attributes to attributes
+        # Therefore, it doesn't seem useful to call it.
+        # pylint: disable=super-init-not-called
```

Comment:
My Python knowledge is a bit lacking here, but wouldn't this mean that a `DefaultSpan` wouldn't have `.attributes` defined? And I thought the point of adding the `__init__` in the `Span` class was to ensure that `.attributes` is always defined?

Reply:
This is a good point. I'll look into the best way to add this to the DefaultSpan object.

Reply:
I'm guessing you can probably just call `super().__init__()` here, which would also remove the lint warning.